src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const minLongPeriod = 7;
+  const minMediumPeriod = 3;
+  const minLongPeriodDiscount = 50;
+  const minMediumPeriodDiscount = 20;
+  const priceADay = 40;
+
+  if (days >= minLongPeriod) {
+    return days * priceADay - minLongPeriodDiscount;
+  }
+
+  if (days >= minMediumPeriod) {
+    return days * priceADay - minMediumPeriodDiscount;
+  } else {
+    return days * priceADay;
+  }
```

Comment:
`else` is useless with `return` inside `if` block.

```suggestion
  if (days >= minMediumPeriod) {
    return days * priceADay - minMediumPeriodDiscount;
  }
 
   return days * priceADay;
```